core/config/config.go

```diff
@@ -160,6 +160,19 @@ func deleteFromPG(ctx context.Context, db pg.DB) error {
 	return errors.Wrap(err, "deleting config stored in postgres")
 }
 
+// Loads config status from sinkdb to see if other raft nodes have configured
+// so that uncofigured nodes can update.
+func UpdateConfigStatus(ctx context.Context, sdb *sinkdb.DB) (*Config, error) {
```

Comment:
This sounds like we're actually updating the config status, which we're not--just polling to see if the config exists. Maybe let's rename it to `CheckConfigExists`? (That's more consistent with the caller's name, too.)